src/main/java/lesson12Person/Main.java

```diff
@@ -0,0 +1,51 @@
+package lesson12Person;
+
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Set<Person> personSetByAge = new TreeSet<>();
```

Comment:
Widzimy, że to Set po typie, więc w nazwie nie musimy tego dawać.
personSetByAge -> peopleByAge

Reply:
Analogicznie w pozostałych przypadkach.

Reply:
poprawione